src/trio/_subprocess_platform/__init__.py

```diff
@@ -106,12 +106,12 @@ def create_pipe_from_child_output():
 
         from .._windows_pipes import PipeReceiveStream, PipeSendStream
 
-        def create_pipe_to_child_stdin():
+        def create_pipe_to_child_stdin() -> tuple[PipeSendStream, int]:
             # for stdin, we want the write end (our end) to use overlapped I/O
             rh, wh = windows_pipe(overlapped=(False, True))
             return PipeSendStream(wh), msvcrt.open_osfhandle(rh, os.O_RDONLY)
 
-        def create_pipe_from_child_output():
+        def create_pipe_from_child_output() -> tuple[PipeReceiveStream, int]:
```

Comment:
there's a mypy setting for catching this as well

Reply:
If you are referring to `disallow_incomplete_defs` and `check_untyped_defs`, those two are already enabled for Trio.

Reply:
I meant `disallow_untyped_defs`

Reply:
`disallow_untyped_defs` is also already enabled

Reply:
then why did this not get caught by mypy previously??

Reply:
Not really sure